tests/test_framework/electrs.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+
+from ephemeral_port_reserve import reserve
+from test_framework.utils import BitcoinBackend, TailableProc, ELECTRS_PATH
+
+
+class Electrs(BitcoinBackend):
```

Comment:
Just repeating a comment from Mattermost, somehow we cannot run tests in parallel anymore with an Electrs backend. This should be fixed. Maybe an issue with the maximum number of concurrent connections possible?

Reply:
i do not find any config about this in electrs and the number of connections [seems not limited](https://github.com/romanz/electrs/blob/9cdb4b90d5613ab1c566e8ff7bcba80ef90b5b7b/src/server.rs#L212-L225) but maybe [about bitcoind](https://github.com/romanz/electrs/blob/master/doc/upgrading.md?plain=1#L330)?

Reply:
Yeah neither did i find the option. But it's curious we wouldn't be able to create multiple connections to it.

Reply:
This is now fixed.